modules/osdk-updating-projects.adoc

```diff
@@ -0,0 +1,353 @@
+ENVTEST ?= $(LOCALBIN)/setup-envtest
+
+## Tool Versions
+KUSTOMIZE_VERSION ?= v3.8.7
+CONTROLLER_TOOLS_VERSION ?= v0.8.0
+
+KUSTOMIZE_INSTALL_SCRIPT ?= "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"
+.PHONY: kustomize
+kustomize: $(KUSTOMIZE) ## Download kustomize locally if necessary.
+$(KUSTOMIZE): $(LOCALBIN)
+	curl -s $(KUSTOMIZE_INSTALL_SCRIPT) | bash -s -- $(subst v,,$(KUSTOMIZE_VERSION)) $(LOCALBIN)
+
+.PHONY: controller-gen
+controller-gen: $(CONTROLLER_GEN) ## Download controller-gen locally if necessary.
+$(CONTROLLER_GEN): $(LOCALBIN)
+	GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-tools/cmd/controller-gen@$(CONTROLLER_TOOLS_VERSION)
+
+.PHONY: envtest
+envtest: $(ENVTEST) ## Download envtest-setup locally if necessary.
+$(ENVTEST): $(LOCALBIN)
+	GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
+---- 
+endif::[]
+
+ifdef::golang,hybrid[]
+.. Update `ENVTEST_K8S_VERSION` and `controller-gen` fields in your `Makefile` to support Kubernetes 1.24:
++
+[source,terminal]
+----
+...
+ENVTEST_K8S_VERSION = 1.24 <1>
+...
+sigs.k8s.io/controller-tools/cmd/controller-gen@v0.9.0 <2>
+----
+<1> Update version `1.22` to `1.24`.
+<2> Update version `0.7.0` to `0.9.0`.
+endif::[]
+
+.. Apply the changes to your `Makefile` and rebuild your Operator by entering the following command:
++
+[source,terminal]
+----
+$ make
+----
+
+ifdef::golang,hybrid[]
+. Make the following changes to the `go.mod` file to update Go and its dependencies:
++
+[source,golang]
+----
+go 1.18 <1>
+
+require (
+  github.com/onsi/ginkgo v1.16.5 <2>
+  github.com/onsi/gomega v1.18.1 <3>
+  k8s.io/api v0.24.0 <4>
+  k8s.io/apimachinery v0.24.0 <4>
+  k8s.io/client-go v0.24.0 <4>
+  sigs.k8s.io/controller-runtime v0.12.1 <5>
+)
+----
+<1> Update version `1.16` to `1.18`.
+<2> Update version `v1.16.4` to `v1.16.5`.
+<3> Update version `v1.15.0` to `v1.18.1`.
+<4> Update version `v0.22.1` to `v0.24.0`.
+<5> Update version `v0.10.0` to `v0.12.1`.
+endif::golang,hybrid[]
+
+ifdef::hybrid[]
+. Update the Helm Operator plug-ins:
++
+[source,golang]
+----
+github.com/operator-framework/helm-operator-plugins v0.0.11 <1>
+----
+<1> Update version `v0.0.8` to `v0.0.11`.
+endif::[]
+
+ifdef::golang,hybrid[]
+. Download and clean up the dependencies by entering the following command:
++
+[source,terminal]
+----
+$ go mod tidy
+----
+endif::[]
+
+ifdef::golang[]
+. If you use `api/webhook_suitetest.go` and `controllers/suite_test.go` suite test files, make the following changes:
++
+.Old suite test file
+[source,golang]
+----
+cfg, err := testEnv.Start()
+----
++
+.New suite test file
+[source,golang]
+----
+var err error
+// cfg is defined in this file globally.
+cfg, err = testEnv.Start()
+----
+
+. If you use the Kubernetes declarative plug-in, update your Dockerfile with the following changes:
+
+.. Add the following changes below the line that begins `COPY controllers/ controllers/`:
```

Comment:
Just checking, are they adding the following lines, or making changes to existing lines? If the former, wondering if it should say "Add the following lines..."

Same for the next step, if so